src/app/badges/badges-page/badges-page.component.html

```diff
@@ -1 +1,23 @@
-<p>badges works!</p>
+<mat-grid-list class="dashboard-container" cols="3"  rowHeight="fit" gutterSize="15px" [style.height]="'100%'">
```

Comment:
it is not a good idea to set height using `[style.height]="'100%'"`
better provide `height: 100%` in `dashboard-container` class.